articles/mysql/flexible-server/how-to-deploy-on-azure-free-account.md

```diff
@@ -125,22 +125,22 @@ Now that you've now created an Azure Database for MySQL flexible server instance
 
 ## Monitor and track free services usage
 
-You're not charged for Azure Database for MySQL flexible server services included for free with your Azure free account unless you exceed the free service limits. To remain within the limits, use the Azure portal to track and monitor your free services usage.
+You're not charged for Azure Database for MySQL Flexible Server services included for free with your Azure free account unless you exceed the free service limits. To remain within the limits, use the Azure portal to track and monitor your free services usage.
 
 1. In the Azure portal, search for **Subscriptions** and select the Azure Free Account - **Free Trial** subscription.
 
 1. On the **Overview** page, scroll down to show the tile **Top free services by usage**, and then select **View all free services**.
 
     :::image type="content" source="media/how-to-deploy-on-azure-free-account/free-services-usage-overview.png" alt-text="Screenshot that shows the Free Trial subscription overview page and highlights View all free services." lightbox="media/how-to-deploy-on-azure-free-account/free-services-usage-overview.png":::
 
-1. Locate the meters related to **Azure Database for MySQL – Flexible Server** to track usage.
+1. Locate the meters related to **Azure Database for MySQL ? Flexible Server** to track usage.
```

Comment:
```suggestion
1. Locate the meters related to **Azure Database for MySQL - Flexible Server** to track usage.
```

Reply:
@ShannonLeavitt - Thanks for the suggestion.